### PR TITLE
chore: bump fluvio to v0.11.11

### PR DESCRIPTION
**Pull request by fraidev**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,24 @@
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
-## Platform Version 0.11.11 - UNRELEASED
+## Platform Version 0.11.11 - 2024-08-05
+
+### Added
+
+* Sdf-hub support updates ([#4124](https://github.com/infinyon/fluvio/issues/4124))
+
```
Comment (digikata) [inline CHANGELOG.md:10]: Can you add a Line for #4122 here?  It looks like git cliff did not automatically pick up "feat: *" commits.

**Review by digikata — APPROVED** [review]
Overall LGTM,  need one manual add (and we should add to the git-cliff regex post release too)

File: CHANGELOG.md (hunk 1 above)
Comment (fraidev) [reply CHANGELOG.md:10]: Sure! added it

Comment (digikata) [reply CHANGELOG.md:11]: Great, LGTM, ready to merge

**Pull request merged**